include/aspect/simulator.h

```diff
@@ -799,10 +817,15 @@ namespace aspect
       /**
        * Invert the action of the function above.
        *
+       * This function modifies @p vector in-place and uses a second copy with relevant
+       * dofs (@p relevant_vector) for accessing the pressure values. Both @p vector and @p relevant_vector are expected to already contain
```

Comment:
I do not get the meaning of the second copy, maybe it is just me :smirk:. Could you elaborate a bit?